src/redux/configureStore.js

```diff
@@ -0,0 +1,11 @@
+import Redux, { combineReducers } from 'react-redux';
+import { booksReducer } from './books/books';
+import { categoriesReducer } from './categories/categories';
+
+const rootReducer = combineReducers({
+  books: booksReducer,
+  categories: categoriesReducer,
+});
+
+const store = Redux.createStoreHook(rootReducer);
+export default store;
```

Comment:
- [ ] I would recommend using the createStore and importing it from the redux and later Uses `createStore` to create a store through the reducer created in the above step.

`import { combineReducers, createStore, applyMiddleware } from 'redux';`